Newsletters: add 188 (2022-02-23) (#724)
bitcoinops/bitcoinops.github.io: _topics/en/fee-sponsorship.md

      link: https://example.com

## Optional.  Each entry requires "title" and "url".  May also use "feature:
## true" to bold entry and "date"
optech_mentions:
  - title: Transaction fee sponsorship proposal
    url: /en/newsletters/2020/09/23/#transaction-fee-sponsorship

  - title: "Call for topics in layer-crossing workshop, including fee sponsorship"
    url: /en/newsletters/2021/04/28/#call-for-topics-in-layer-crossing-workshop

  - title: "Fee accounts, an outgrowth from fee sponsorship"
    url: /en/newsletters/2022/01/12/#fee-accounts

  - title: Fee bumping discussion with advocacy for fee sponsorship
    url: /en/newsletters/2022/02/23/#fee-bumping-and-transaction-fee-sponsorship

## Optional.  Same format as "primary_sources" above
see_also:
  - title: Child Pays For Parent
    link: topic cpfp

  - title: Transaction pinning
    link: topic transaction pinning

## Optional.  Force the display (true) or non-display (false) of stub
## topic notice.  Default is to display if the page.content is below a
## threshold word count
#stub: false

## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
## Should be less than 500 characters
excerpt: >

>   **Fee sponsorship** is the proposed ability to allow creation of

Reviewer: ```suggestion
  **Fee sponsorship** is a proposed consensus change to allow creation of
```

and link? https://gist.github.com/JeremyRubin/92a9fc4c6531817f66c2934282e71fdf

  transactions which effectively add fees to other unrelated
  transactions, possibly helping those other transactions confirm

>   faster.  It's similar to CPFP but without the transactions needing to

Reviewer: ```suggestion
  faster.  It enables CPFP but without the transactions needing to
```
